### PR TITLE
Dp 7144/search tabs autosuggest

### PR DESCRIPTION
**Pull request by royderks**

### CODE REVIEW

File: src/header/containers/header-search/HeaderSearch.jsx [hunk 1]
```diff
@@ -71,21 +77,41 @@ class HeaderSearch extends React.Component {
...
 
     if (activeSuggestion.index === -1) {
       // Load the search results
       onCleanDatasetOverview() // TODO, refactor: don't clean dataset on search
-      if (toDataset) {
-        onDatasetSearch(typedQuery)
+
+      const searchType =
+        label ||
+        // eslint-disable-next-line no-nested-ternary
```
Comment (chrisvanmook) [inline src/header/containers/header-search/HeaderSearch.jsx]: Just a personal style thing, perhaps move the eslint-disable to the top to disable the rule for the whole file and destructure the constants?

File: src/header/containers/header-search/HeaderSearch.jsx [hunk 2]
```diff
@@ -6,6 +6,12 @@ import { extractIdEndpoint } from '../../../store/redux-first-router/actions'
 import useSlug from '../../../app/utils/useSlug'
 import { VIEW_MODE } from '../../../shared/ducks/ui/ui'
 
+export const CONSTANTS = {
```
Comment (chrisvanmook) [inline src/header/containers/header-search/HeaderSearch.jsx]: don't we use this somewhere else as well?

File: src/header/containers/header-search/HeaderSearch.jsx (hunk 1 above)
Comment (royderks) [reply src/header/containers/header-search/HeaderSearch.jsx]: Sure!

File: src/header/containers/header-search/HeaderSearch.jsx (hunk 2 above)
Comment (royderks) [reply src/header/containers/header-search/HeaderSearch.jsx]: Yes, imported them from the `app/routes.js`

**Pull request merged**